### ISSUE TITLE
Add tests for power op

### ISSUE THREAD

**Issue by kmitrovicTT** [tenstorrent/tt-xla#62]
Sub-issue of https://github.com/tenstorrent/tt-mlir/issues/1033

**Comment by kmitrovicTT** [comment]
Blocked by https://github.com/tenstorrent/tt-mlir/issues/1094

**Comment by skhatriTT** [comment]
Hey @mrakitaTT do you happen to know if this is still ongoing?

**Comment by mrakitaTT** [comment]
@skhatriTT yes, @kmitrovicTT has implemented the op conversion in tt-mlir, now we have to add tests in tt-xla. This (and similar OP test issues) was on our backlog with lower priority, waiting to get another contractor to start working on this. We got one just yesterday, once they onboard they will work on this.
